### PR TITLE
A temp solution for locked problems.

### PR DESCRIPTION
**Pull request by sykuang**
Simply disable the user checking and download locked issue directly. works on both premium and normal account.

### CODE REVIEW

File: lib/plugins/leetcode.js [hunk 1]
```diff
@@ -120,7 +120,7 @@ plugin.getCategoryProblems = function(category, cb) {
 plugin.getProblem = function(problem, cb) {
   log.debug('running leetcode.getProblem');
   const user = session.getUser();
-  if (problem.locked && !user.paid) return cb('failed to load locked problem!');
+  //if (problem.locked && !user.paid) return cb('failed to load locked problem!');
```
Comment (skygragon) [inline lib/plugins/leetcode.js]: for non-premium user, what's the error now if fetching a locked question?

Comment (sykuang) [reply lib/plugins/leetcode.js]: It will show "[ERROR] failed to load problem! [0]" when non-premium user try to access locked problem.

**Comment by skygragon** [comment]
@sykuang would you mind trying https://github.com/skygragon/leetcode-cli/commit/fc7a09b7efcdb1cc3a8bdd88b857c7bc1f8cc078 to see if it works for premium user?

File: lib/plugins/leetcode.js (hunk 1 above)
Comment (wilbeibi) [reply lib/plugins/leetcode.js]: Cool, this works for me

**Comment by fzxt** [comment]
@skygragon Just tried after updating latest, works!

**Pull request merged**